ui/admin/tests/integration/components/worker-diagram/dual-filter/enterprise/index-test.js

```diff
@@ -5,17 +5,33 @@ import { hbs } from 'ember-cli-htmlbars';
 import { setupIntl } from 'ember-intl/test-support';
 
 module(
-  'Integration | Component | worker-diagram/dual-filter',
+  'Integration | Component | worker-diagram | dual-filter | enterprise',
   function (hooks) {
     setupRenderingTest(hooks);
     setupIntl(hooks);
 
+    let featuresService;
+    const targetWorkerFilterIngress = 'target-worker-filters-v2-ingress';
+    const targetWorkerFilterHCP = 'target-worker-filters-v2-hcp';
+
+    hooks.beforeEach(function () {
+      featuresService = this.owner.lookup('service:features');
+      featuresService.enable(targetWorkerFilterIngress);
+      featuresService.disable(targetWorkerFilterHCP);
+    });
+
     test('it renders the correct diagram when egressFilter is false and ingress is false', async function (assert) {
-      assert.expect(4);
+      assert.expect(6);
       await render(
         hbs`<WorkerDiagram::DualFilter @egressFilter={{false}} @ingressFilter={{false}} />`
       );
 
+      assert
+        .dom('.ordered-series-diagram-item:nth-child(2)')
+        .hasText('Any worker');
+      assert
+        .dom('.ordered-series-diagram-item:nth-child(2)')
+        .doesNotIncludeText('HCP worker');
```

Comment:
This is one of the asserts that we could remove once the `WorkerDiagram::DualFilter` has been split into 2 new components.